### PR TITLE
Request system link

### LINKED ISSUE #379: Navigate to System page from Command and Request Views
It would be helpful to have  the system+version definition on the command view page be a link to that system page.

That way if you click "Pour it again" and then need to return to the command listing for that System it's really easy.

### PR DESCRIPTION
**Pull request by west270**
Added hyperlink in request view to system view.
Fixes #379

### CODE REVIEW

File: src/ui/src/partials/request_view.html [hunk 1]
```diff
@@ -49,7 +49,11 @@ <h1 class="page-header">
                   ng-class="{'chevron-rotate': !childrenCollapsed}">
             </span>{{data.command}}</td>
           <td>{{data.metadata.system_display_name || data.system}}</td>
-          <td>{{data.system_version}}</td>
+          <td><div style="display: inline-block; padding-top: 3px; padding-right: 16px;">
+              <a ui-sref="base.namespace.system({systemName: data.system, systemVersion: data.system_version})">
+                <span style="...">{{data.system_version}}</span>
+              </a>
+            </div></td>
```
Comment (hazmat345) [inline src/ui/src/partials/request_view.html]: Minor nitpick - the tags here should be nested and indented.

```html
<td>
  <div>
...
  </div>
</td>

File: src/ui/src/partials/request_view.html [hunk 2]
```diff
@@ -49,7 +49,11 @@ <h1 class="page-header">
                   ng-class="{'chevron-rotate': !childrenCollapsed}">
             </span>{{data.command}}</td>
           <td>{{data.metadata.system_display_name || data.system}}</td>
-          <td>{{data.system_version}}</td>
+          <td><div style="display: inline-block; padding-top: 3px; padding-right: 16px;">
+              <a ui-sref="base.namespace.system({systemName: data.system, systemVersion: data.system_version})">
+                <span style="...">{{data.system_version}}</span>
```
Comment (hazmat345) [inline src/ui/src/partials/request_view.html]: Is the style tag needed here?

File: src/ui/src/js/controllers/request_index.js [hunk 3]
```diff
@@ -164,7 +164,12 @@ export default function requestIndexController(
       }),
     DTColumnBuilder
       .newColumn('system_version')
-      .withTitle('Version'),
+      .withTitle('Version')
+      .renderWith(function(data, type, full) {
+        let display = '';
+
+        return display + `<a ui-sref="base.namespace.system({systemName: '${full.system}', systemVersion: '${full.system_version}'})">` + data + '</a>';
```
Comment (hazmat345) [inline src/ui/src/js/controllers/request_index.js]: Is the variable here needed? Could we just do:

```javascript
return `<a ui-sref="base.namespace.system(...
```

Comment (west270) [reply src/ui/src/js/controllers/request_index.js]: I made requested changes and added link for command view.

**Pull request merged**